index.html

```diff
@@ -459,6 +459,7 @@ <h4>Workspace</h4>
                                 <option value="1">Freq. vs Throttle</option>
                                 <option value="3">Freq. vs RPM</option>
                                 <option value="2">Error vs Setpoint</option>
+                                <option value="4">PSD vs frequency</option>
```

Comment:
PSD is already in frequency domain. also in other plots it is x vs y, while here frequency is on the x axis. so maybe this way would be better, or "frequency vs noise level"
```suggestion
                                <option value="4">Power spectral density</option>
```